cmake/install.cmake

```diff
@@ -87,19 +95,34 @@ ELSE()
   SET(INSTALL_PLUGINDIR_RPM "lib/mariadb/plugin")
 ENDIF()
 SET(INSTALL_INCLUDEDIR_RPM "include")
-SET(INSTALL_DOCDIR_RPM "docs")
+SET(INSTALL_DOCDIR_RPM "share/doc/mariadb-connector-odbc")
+SET(INSTALL_LICENSEDIR_RPM ${INSTALL_DOCDIR_RPM})
 SET(LIBMARIADB_STATIC_RPM "mariadbclient")
 
 #
 # DEB layout
-#
+# Only ia-32 and amd64 here. the list is too long to hardcode it
+IF(NOT CMAKE_LIBRARY_ARCHITECTURE) 
+  IF(CMAKE_SIZEOF_VOID_P EQUAL 8)
+    SET(CMAKE_LIBRARY_ARCHITECTURE "x86_64-linux-gnu")
+  ELSE()
+    SET(CMAKE_LIBRARY_ARCHITECTURE "i386-linux-gnu")
+  ENDIF()
+ENDIF()
+
 SET(INSTALL_BINDIR_DEB "bin")
 SET(INSTALL_LIBDIR_DEB "lib/${CMAKE_LIBRARY_ARCHITECTURE}")
 SET(INSTALL_PCDIR_DEB "lib/${CMAKE_LIBRARY_ARCHITECTURE}/pkgconfig")
 SET(INSTALL_PLUGINDIR_DEB "${INSTALL_LIBDIR_DEB}/libmariadb${CPACK_PACKAGE_VERSION_MAJOR}/plugin")
 SET(INSTALL_INCLUDEDIR_DEB "include/mariadb")
+SET(INSTALL_DOCDIR_DEB "share/doc/mariadb-connector-odbc")
+SET(INSTALL_LICENSEDIR_DEB "${INSTALL_DOCDIR_DEB}")
 SET(LIBMARIADB_STATIC_DEB "mariadb")
 
+IF(INSTALL_LAYOUT MATCHES "RPM")
+  SET(INSTALL_PLUGINDIR_CLIENT ${INSTALL_PLUGINDIR_RPM})
+ENDIF()
+
 IF(INSTALL_LAYOUT MATCHES "DEB")
```

Comment:
if you move this few pages down, you won't need an IF.

Reply:
not sure I understand. there are 3 possible values to set INSTALL_PLUGINDIR_CLIENT  depending on install layout. 
both 3 are present now and RPM one was missing:
INSTALL_PLUGINDIR_DEFAULT
INSTALL_PLUGINDIR_DEB
INSTALL_PLUGINDIR_RPM

Reply:
At the end of the file, `INSTALL_PLUGINDIR` will be set to `INSTALL_PLUGINDIR_${INSTALL_LAYOUT}`
After that you can simply do
```
SET(INSTALL_PLUGINDIR_CLIENT ${INSTALL_PLUGINDIR})
```
with no conditions on the `INSTALL_LAYOUT`

Reply:
no, that's not true. in this case INSTALL_PLUGINDIR and INSTALL_PLUGINDIR_CLIENT will be filled with default value from CMakeLists.txt

Reply:
fixed by removing duplicated logic from CMakeLists.txt